packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix-preview.vue

```diff
@@ -46,6 +49,7 @@ const subject = ref('');
 const outcome = ref('');
 
 const matrixColLen = computed(() => matrix.value[0]?.length ?? 0);
+const matrixThreshold = 20;
```

Comment:
Do we not usually have a convention of all upper case letters for const primitives? It's ok if not. Just asking

Reply:
fixed